openwisp_monitoring/device/admin.py

```diff
@@ -136,9 +136,15 @@ class Media:
             )
```

Comment:
Notice range and date: 
![vm1](https://user-images.githubusercontent.com/24985760/184555243-c5941d88-aac6-4311-80b5-11e33c57f05b.png)

Reply:
> Notice range and date: ![vm1](https://user-images.githubusercontent.com/24985760/184555243-c5941d88-aac6-4311-80b5-11e33c57f05b.png)

This is because the data was not set in local storage as the previous one and hence after getting the chart on refresh the data is the previous one. Now I have improved this.